docker/lnd/start-lnd.sh

```diff
@@ -42,16 +42,17 @@ set_default() {
 RPCUSER=$(set_default "$RPCUSER" "devuser")
 RPCPASS=$(set_default "$RPCPASS" "devpass")
 DEBUG=$(set_default "$DEBUG" "debug")
-BITCOIN_NETWORK=$(set_default "$BITCOIN_NETWORK" "simnet")
+NETWORK=$(set_default "$NETWORK" "simnet")
+CHAIN=$(set_default "$CHAIN" "bitcoin")
 
 lnd \
     --datadir="/data" \
     --logdir="/data" \
-    --bitcoin.active \
-    "--bitcoin.$BITCOIN_NETWORK" \
-    --bitcoin.rpchost="btcd" \
-    --bitcoin.rpccert="/rpc/rpc.cert" \
-    --bitcoin.rpcuser="$RPCUSER" \
-    --bitcoin.rpcpass="$RPCPASS" \
+    "--$CHAIN.rpccert"="/rpc/rpc.cert" \
+    "--$CHAIN.active" \
+    "--$CHAIN.$NETWORK" \
+    "--$CHAIN.rpchost"="blockchain" \
```

Comment:
Actually scratch that. Not sure what happened before, but I'm able to properly connect now! Once the minor doc comments are addressed, I'll get this merged.